src/main/java/org/veupathdb/service/sequence/controller/SequenceController.java

```diff
@@ -0,0 +1,39 @@
+package org.veupathdb.service.sequence.controller;
+
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.Request;
+import jakarta.ws.rs.core.StreamingOutput;
+
+import org.veupathdb.lib.container.jaxrs.server.annotations.Authenticated;
+import org.veupathdb.service.sequence.generated.model.SequencePostRequest;
+import org.veupathdb.service.sequence.generated.resources.SequenceSequenceType;
+
+import org.veupathdb.service.sequence.util.StreamSequences;
+import org.veupathdb.service.sequence.util.Validate;
+import org.veupathdb.service.sequence.service.Sequences;
+
+public class SequenceController implements SequenceSequenceType {
+
+  @Context
+  private Request req;
+
+  @Override
+  @Authenticated
```

Comment:
Curious what part of this requires a valid user?

Reply:
Nothing does! @Authenticated is not neccessary.